src/services/timerFunction.ts

```diff
@@ -0,0 +1,214 @@
+import moment, { Moment } from "moment";
+import {
+  addFinishedExecution,
+  addOngoingExecution,
+  getOngoingExecutions,
+  setOngoingExecution,
+  removeOngoingExecution,
+} from "./localStorage";
+import { ExecutionStatus, OngoingExecution, FinishedExecution } from "./types";
+
+/**
+ * Coordena o a manipulação de execuções.
+ *
+ * O objetivo é centralizar as funções relacionadas à execuções, de forma a
+ * não poluir as Screens que em que serão utilizadas e que mudanças necessárias
+ * sejam automaticamente propagadas a todas as telas que necessitem trabalhar com
+ * execuções.
+ *
+ */
+
+/** Função responsável por criar uma nova execução */
+export const createExecution = async (cardInfo: any) => {
```

Comment:
Cria um tipo (em `types`) pra especificar os dados que são necessários neste cardInfo, acho que vai facilitar pra depois

```suggestion
export const createExecution = async (cardInfo: CardExecutionType ) => {
```